registry-requirements.md

```diff
@@ -120,7 +120,19 @@ What does the binding have to contain to go into the table
     - Implementing a non-commercial Consumer application/driver
     - Conditions for commercial use: E.g. building a commercial product with the binding
     - Making a statement about your product's supporting that binding
-9. DISCUSS: What is the objective mechanism to confirm the status change from "initial" to "current"? Should we have a test suite? Plugfest experience?
+9. Transition from "Initial" to "Current"
+  - Starting from the initial submission each binding has to demonstrate a certain level of concrete development maturity. This process involves real-world testing, which can take place in Plugfests, independent testing events, or even informal collaboration between developers. These testing events do not have to be organized by W3C and can be conducted remotely, including over VPN. The goal is to demonstrate that the binding correctly maps protocol operations and is well understood by at least two parties.
+  - At each testing event every operation defined in the binding MUST be validated automatically (e.g., scripts, test suites, etc.) and the results SHOULD be published in a dedicated document (README, or other human-readable documents) called *Test Report*.
```

Comment:
> the results SHOULD be published in a dedicated document

There is a small logical gap. We do not mandate the existence of a test report anywhere. So an event doesn't need a test report output but at least one test report should exist in the end.